scs-multiapi-maven-plugin/pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>com.sngular</groupId>
   <artifactId>scs-multiapi-maven-plugin</artifactId>
-  <version>5.3.1</version>
+  <version>5.3.2</version>
```

Comment:
You are missing change version number in 3 more files...

Reply:
fixed